app/_includes/prereqs/tools/konnect-api.md

```diff
@@ -0,0 +1,7 @@
+To use the copy, paste, and run the instructions in this how-to, you must export these additional environmental variables:
+
+```sh
+export KONNECT_API_REGION_URL=https://{region}.api.konghq.com
+export CONTROL_PLANE_ID=your-control-plane-uuid
+export EXAMPLE_ROUTE_ID=your-example-route-id
+```
```

Comment:
These types of includes that get pulled in automatically need some extra structure, like this:

```
{% capture details_content %}
To use the copy, paste, and run the instructions in this how-to, you must export these additional environmental variables:

```sh
export KONNECT_API_REGION_URL=https://{region}.api.konghq.com
export CONTROL_PLANE_ID=your-control-plane-uuid
export EXAMPLE_ROUTE_ID=your-example-route-id
```
{% endcapture %}

{% include how-tos/prereq_cleanup_item.html summary='Admin API' details_content=details_content icon_url='/assets/icons/code.svg' %}
```

Reply:
Thanks! I moved this to an inline prereq instead so that it would show after the example entities.